app/styles/_onboarding.scss

```diff
@@ -0,0 +1,158 @@
+// VARS
```

Comment:
Which of these vars are being used in your changes and which aren't? Delete the ones that aren't